content/components/border-box.mdx

```diff
@@ -7,9 +7,11 @@ railsIds:
 
 import ComponentLayout from '~/src/layouts/component-layout'
 import {Text} from '@primer/react'
+import {Link, Text} from '@primer/react'
+import {Link as GatsbyLink} from 'gatsby'
 export default ComponentLayout
 
 <Note variant="warning">
   <Text sx={{display: 'block', fontWeight: 'bold', mb: 2}}>Work in progress</Text>
-  We are currently working on general guidance for this component.
+  Interested in contributing? Review the guidelines for <Link as={GatsbyLink} to="/guides/contribute/documentation#documenting-components">writing component documentation</Link> and open a pull request in our <Link as={GatsbyLink} to="https://github.com/primer/design">documentation repository.</Link>
 </Note>
```

Comment:
This text is different from the "work in progress" messaging on other components. I think we should keep it consistent.

Reply:
It might be worth turning this whole thing into it's own component so we don't have to copy/paste all this content in multiple files.